src/Drivers/Translation.php

```diff
@@ -78,7 +78,7 @@ public function autoTranslate($language = false)
      * @throws \ErrorException
      */
     public function getGoogleTranslate($language,$token){
-        $tr = new GoogleTranslate($language, $this->sourceLanguage);
+        $tr = new GoogleTranslate($language);
```

Comment:
It isn't strictly necessary to pass the source language, but it's my understanding that if you don't, Google Translate will try to guess the source language (it's set to `auto`). That can sometimes lead to wrong results if two languages have the same word with a different meaning, e.g. the word 'brand' means fire in Dutch.

Reply:
oh I see. that is correct. 
I didn't look at that.

Reply:
No problem. Thanks for your part of writing this pull request

Reply:
It is my pleasure. 
It was quite helpful to me and I thought to let others use it also